tests/integration/conftest.py

```diff
@@ -72,6 +72,33 @@ def _pytest_passed(request):
     ))
 
 
+def _rmtree(path, tries=5):
+    """Remove a directory and its contents.
+
+    Attempt to remove a directory up to ``tries`` times before failing.
+
+    This should be used in-place of ``shutil.rmtree`` which doesn't
+    work reliably on NFS.
+
+    See also:
+        * https://stackoverflow.com/questions/58943374/
+          shutil-rmtree-error-when-trying-to-remove-nfs-mounted-directory
+        * https://bugzilla.redhat.com/show_bug.cgi?id=1362667
+
+    Args:
+        path (pathlib.Path): The directory to remove.
+        tries (int): Maximum number of attempts to make.
+
+    """
+    for _ in range(tries):
+        try:
+            rmtree(path)
+            return
+        except OSError as exc:
+            pass
+    raise exc
```

Comment:
Still getting scope problems with this... think this fixes it... 
```suggestion
    """
    outer_exc = None
    for _ in range(tries):
        try:
            rmtree(path)
            return
        except OSError as exc:
            outer_exc = exc
            pass
    raise outer_exc
```